### PR TITLE
Added redirect map for unity.ac hosts

### PR DESCRIPTION
**Pull request by davidoae**
Added specific redirect for Vienna, so should be merged when their host is updated.
Tested in qa.

### CODE REVIEW

File: modules/nginx/manifests/init.pp [hunk 1]
```diff
@@ -100,6 +100,20 @@
...
+    ##################
+
+    file { "${nginx_conf_dir}/redirect_map.conf":
+        ensure  => present,
+        mode    => 0640,
+        owner   => $owner,
+        group   => $group,
+        source  => "puppet:///modules/nginx/redirect_map.conf",
+        require => $nginx_config_requires,
+    }
```
Comment (mrvisser) [inline modules/nginx/manifests/init.pp]: There's a potential bootstrapping issue here in that the service could be installed and started before the redirect_map.conf file is placed. I think that nginx service should have this file added to its `require` dependency list.

**Comment by mrvisser** [comment]
QA0 seems to fail with this:

```
 * Stopping Nginx Server...
   ...fail!
 * Starting Nginx Server...
nginx: [emerg] could not build the map_hash, you should increase map_hash_bucket_size: 32
   ...done.
```

However QA2 seems fine. All the nginx versions *appear* to be the same across machines. Can you first look into this error and why nginx on QA0 would fail but not QA2?

File: modules/nginx/manifests/init.pp (hunk 1 above)
Comment (davidoae) [inline modules/nginx/manifests/init.pp]: good call

**Comment by mrvisser** [comment]
Looks good.

**Pull request merged**